src/main/java/br/edu/ufcg/computacao/eureca/common/exceptions/CommonExceptionHandler.java

```diff
@@ -60,6 +60,6 @@
     public final ResponseEntity<ExceptionResponse> handleAnyException(Exception ex, WebRequest request) {
 
         ExceptionResponse errorDetails = new ExceptionResponse(ex.getMessage(), request.getDescription(false));
-        return new ResponseEntity<>(errorDetails, HttpStatus.UNSUPPORTED_MEDIA_TYPE);
+        return new ResponseEntity<>(errorDetails, HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
This error code should never be returned. When it is returned it identifies a bug in our code (a generic exception that has not been caught and transformed into one of the known Eureca exceptions handled in the previous calls). If you change to INTERNAL_SERVER_ERROR when the user reports the problem, the support/development team won't be able to distinguished whether an InternalServerErrorException was raised (no problem with our code) or any other exception was raised (a problem with the code).

Reply:
Makes sense